src/chainstate/stacks/db/mod.rs

```diff
@@ -1056,6 +1056,19 @@ pub mod test {
         StacksChainState::open(mainnet, chain_id, &path).unwrap()
     }
 
+    pub fn instantiate_chainstate_with_balances(mainnet: bool, chain_id: u32, test_name: &str, balances: Vec<(StacksAddress, u64)>) -> StacksChainState {
```

Comment:
Should this be marked `#[cfg(test)]`?

Reply:
It is -- the module it's declared in is `#[cfg(test)]`.